etc/env/base/fri_proof_compressor.toml

```diff
@@ -1,10 +1,10 @@
 [fri_proof_compressor]
-compression_mode = 1
+compression_mode = 5
 prometheus_listener_port = 3321
 prometheus_pushgateway_url = "http://127.0.0.1:9091"
 prometheus_push_interval_ms = 100
 generation_timeout_in_secs = 3600
 max_attempts = 5
-universal_setup_path = "../keys/setup/setup_2^24.key"
-universal_setup_download_url = "https://storage.googleapis.com/matterlabs-setup-keys-us/setup-keys/setup_2^24.key"
+universal_setup_path = "../keys/setup/"
```

Comment:
well, since we are having different setup keys now, it doesn't make sense to specify the only file. Instead, for now the naming of the file is determined on the side of code, but looking from that perspective I agree that probably additional config values would make more sense